cmd/dockerd/daemon_systemd_unix.go

```diff
@@ -0,0 +1,24 @@
+// +build !nosystemd
+
+package main
+
+import (
+	"github.com/docker/docker/daemon"
+	"github.com/docker/docker/daemon/config"
+	systemdDaemon "github.com/coreos/go-systemd/v22/daemon"
+)
+
+func newCgroupParent(config *config.Config) string {
+	cgroupParent := "docker"
+	useSystemd := daemon.UsingSystemd(config)
```

Comment:
I don't understand why this would need a build-tag, as it's already checking if we're using with systemd?

Reply:
> I don't understand why this would need a build-tag, as it's already checking if we're using with systemd?

much smaller binary by dropping all systemd and dbus related stuff.
reduced complexity in the final binary, less code to audit.